tests/test_pdf_download.py

```diff
@@ -1,25 +1,40 @@
+import pytest
+import requests
+from bs4 import BeautifulSoup
+from pprint import pprint
+
 from .data.pdf_download_data import HTML_TEXT, YEAR_LINKS
 
+from bike_rack.check_page_setup import check_page_setup
```

Comment:
Is the intention that all tests get imported from `bike_rack` until they are initially merged?